packages/cli/src/services/project-parser.ts

```diff
@@ -23,12 +23,12 @@ type ProjectParseOpts = {
   checkDefaults?: CheckConfigDefaults,
   browserCheckDefaults?: CheckConfigDefaults,
   availableRuntimes: Record<string, Runtime>,
+  defaultRuntimeId: string,
   verifyRuntimeDependencies?: boolean,
   checklyConfigConstructs?: Construct[],
 }
 
 const BASE_CHECK_DEFAULTS = {
```

Comment:
nit. this can be removed

Reply:
Yeah I know but I decided to leave it in order to keep the PR a bit simpler, and it could potentially be useful for something.